installer/bundle/bundle_skel.sh

```diff
@@ -830,13 +830,18 @@ if [ "$installMode" = "I" -o "$installMode" = "U" ]; then
     fi
     curl_installed
     if [ $? -ne 0 ]; then
+        echo "curl was not found, attempting to install curl..."
+        install_extra_package curl
+    fi
+    check_program_in_path curl
```

Comment:
That step is unnecessary. I noticed a duplicate code in tar and sed.  We would need to clean that up instead. These are programs agent installer depends on, that is all. whether they show up in package managers or not , that would be an issue with curl installer. In the agent code base we directly call these programs from command line.